docs/source/lda.rst

```diff
@@ -94,4 +94,16 @@ The package provides several functions to perform Linear Discriminant Analysis.
 
     :return: The resultant linear discriminant functional of type ``LinearDiscriminant``.
 
-
+    **Keyword arguments:**
+
+    ============== =============================================================== =========================
+      name                    description                                                          default
+    ============== =============================================================== =========================
+     covestimator   Custom covariance estimator for between-class covariance.          ``SimpleCovariance()``
+                    The covariance matrix will be calculated as
+                    ``cov(covestimator_between, #=data=#; dims=2,
```

Comment:
Sorry for being a pain, but the `#=` syntax looks weird. Why not just use `data` and `...`?

Reply:
No problem, it's a good idea to polish things now 😃 . I've used the comment-style syntax to indicate that `data` is not a meaningful symbol here. Writing just `data` would look like an unexplained symbol and three dots are very similar to the splatting syntax. There should be, I think, a standard notation for 'here goes some expression' in docs.

Reply:
OK, fine with me.